### DISCUSSION TITLE
Multiple GA4 Property IDs on the same install possible?

### DISCUSSION THREAD

**Discussion by JorisOrangeStudio** [freekmurze/laravel-analytics#527] [Q&A]
Hi! I just came across this package. I was wondering if I can also use this on 1 install, with multiple GA4 Property IDs. In the config I can only fill in 1 `property_id`. However, I would like to use this on an install that will show data from multiple websites (with the same Service-Account) . How can I best approach this?

**Comment by acewebsa** [comment]
I was looking into the same thing, and you can set it in the following way:

```
use Spatie\Analytics\Facades\Analytics;
$id = '111111111'; // replace with the property Id of your analytics account
$analytics = Analytics::setPropertyId($id);
dd($analytics);
```
Give this a go with changing the id to each of the id's that you need to use.
Keep in mind you'll have to enable the email address of your google service provider in each property.

**Comment by jegardiner** [comment]
If anyone is trying @acewebsa answer and getting a "no property ID specified" error. You still need to provide a value (even an invalid one) for `ANALYTICS_PROPERTY_ID` in your environment / config, so it can construct the Analytics object. Then you can change the property using the setPropertyID() method.

Otherwise, you'll need to create a new instance of Analytics yourself and pass the property ID in configuration to the contructor.
